src/inputs.jl

```diff
@@ -117,14 +117,16 @@ Base.@kwdef mutable struct InputsStruct
     cop_map_eft_heating_cooling::Array{Any,1} = Dict[]
     wwhp_cop_map_eft_heating::Array{Any,1} = Dict[]
     wwhp_cop_map_eft_cooling::Array{Any,1} = Dict[]
+    
     # Model Settings
-    simulation_years::Int64 = 25  # Number of years for GHP-GHX model
+    hybrid_auto_ghx_sizing_flag::Bool = false
+    simulation_years::Int64 = hybrid_auto_ghx_sizing_flag ? 2 : 25  # Number of years for GHP-GHX model
```

Comment:
Does this way of setting `simulation years` overwrite a user provided `simulation years`?

Reply:
Seems like it would overwrite a user provided value.

Reply:
Updated per your feedback.

Reply:
What did you update?

Reply:
Updated the input defaults so user provided inputs dont get overwritten and moved the automatic hybrid sizing logic to an if block per https://github.com/NREL/GhpGhx.jl/pull/7/commits/25184b9fea90eece740322745394146862f35613

Reply:
Ah for some reason the GitHub interface wasn't showing that change for me. That looks good.